internal/multierror/multierror_test.go

```diff
@@ -0,0 +1,17 @@
+package multierror
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestErrorJoin(t *testing.T) {
+	err := Join(errors.New("foobar"), fmt.Errorf("invalid config:\n%w", Join(errors.New("foo"), errors.New("bar"))))
+	assert.Equal(t, err.Error(), `* foobar
+* invalid config:
+	* foo
+	* bar`)
+}
```

Comment:
It can be useful to define a var or const for the expected result. `assert.Equal` will then include the name of the variable in the failure, which makes it easier to see which one is the _expected_

```suggestion
	const expected = `* foobar
* invalid config: 
	* foo
	* bar`
	assert.Equal(t, err.Error(), expected)
```

Which shows something like;

```console
    multierror_test.go:23: assertion failed: 
        --- ←
        +++ expected
        @@ -1,3 +1,3 @@
        -* foobar
        +* foobarAAA
         * invalid config: 
         	* foo
```